packages/cli-platform-apple/src/tools/installPods.ts

```diff
@@ -51,8 +51,8 @@ async function runPodInstall(loader: Ora, options?: RunPodInstallOptions) {
     if (stderr.includes('pod repo update') && shouldHandleRepoUpdate) {
       await runPodUpdate(loader);
       await runPodInstall(loader, {
+        ...options,
         shouldHandleRepoUpdate: false,
-        newArchEnabled: options?.newArchEnabled,
```

Comment:
Why are you removing this?

Reply:
The `...options: RunPodInstallOptions` includes whether `newArchEnabled` has been configured on / off.

Reply:
Great thanks for the clarification